salt/states/ssh_known_hosts.py

```diff
@@ -12,6 +12,7 @@
         - present
         - user: root
         - fingerprint: 16:27:ac:a5:76:28:2d:36:63:1b:56:4d:eb:df:a6:48
+        - fingerprint_hash_type: md5
```

Comment:
@cachedout I haven't gotten sha256 to work for me, so I'll not add that example and leave this md5 one.

I tried with the github hash of `SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8`, and without the prefix as `nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8`, with and without a line reading `- fingerprint_hash_type: sha256`. If I'm simply doing something wrong, I'd appreciate some else fixing this bit of documentation so I can use sha256 myself. In the meantime I'll have to stick with md5.